pkg/cmd/server/origin/admission/chain_builder.go

```diff
@@ -98,7 +98,7 @@ var (
 		"Priority",
 		"ExtendedResourceToleration",
 		"DefaultTolerationSeconds",
-		"PVCProtection",
+		"StorageObjectInUseProtection",
```

Comment:
how does the admission chain order relate to the one that is used by default in kube? I guess we still need a custom one, but should at least test that our order is an extension of the upstream one.